### PR TITLE
Add canonical tests for Wordy

### PR DESCRIPTION
**Pull request by IanWhitney**
Currently 13 tracks implement this problem. The full list of each track's tests can be found [over at this gist](https://gist.github.com/IanWhitney/3fe2e8ca98d2c6ef37dccb313172f69b)

Of the 13, 11 tracks share the exact same test suite. Those are the tests that I have canonized in this json file.

Python and Scala each have their own tests. But the behavior under test is mostly the same.

Python checks for [some additional failure cases](https://github.com/exercism/xpython/blob/master/exercises/wordy/wordy_test.py#L52)

Scala [allows students to implement powers](https://github.com/exercism/xscala/blob/master/exercises/wordy/src/test/scala/WordProblemTest.scala#L63) as extra credit.

I can see putting some of the Python failure tests in here, but I wanted to get more feedback before making that change.

### CODE REVIEW

**Comment by kytrinyx** [comment]
This looks good to me. If someone else also thinks so, go ahead and merge it.

File: wordy.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            "expected": -11
+      },
+      {
+            "description": "large addition",
+            "input": "What is 123 plus 45678?",
+            "expected": 45801
+      },
+      {
+            "description": "subtraction",
+            "input": "What is 4 minus -12",
```
Comment (petertseng) [inline wordy.json]: question mark or not?

**Comment by petertseng** [comment]
just one comment otherwise :+1:

File: wordy.json (hunk 1 above)
Comment (kytrinyx) [inline wordy.json]: I think we should have the question mark.

**Pull request merged**